config/routes.rb

```diff
@@ -4,4 +4,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :fishes, only: %i[index show new create] do
+    resources :bookings, only: %i[:new :create]
+  end
+  get "pages", to: "pages#about"
```

Comment:
l'url pour la page about doit être comme ça sauf erreur

get **"pages/about"**, to: "pages#about"